tests/models/test_taskinstance.py

```diff
@@ -489,10 +490,14 @@ def test_task_retry_wipes_next_fields(self, session, dag_maker):
         assert ti.next_kwargs is None
         assert ti.state == State.UP_FOR_RETRY
 
-    def test_retry_delay(self, dag_maker):
+    @patch('airflow.utils.timezone.utcnow')
```

Comment:
What do you think about [freezegun](https://github.com/spulec/freezegun)?

Reply:
Cool, looks like exactly what I need!